packages/ubuntu_wsl_setup/lib/main.dart

```diff
@@ -7,11 +7,24 @@ import 'app.dart';
 void main(List<String> args) {
   final options = parseCommandLine(args, onPopulateOptions: (parser) {
     parser.addFlag('reconfigure');
+    parser.addOption(
+      'prefill',
+      valueHelp: 'prefill',
+      help: '''
```

Comment:
I'd suggest also `hide: true` since it's not for users (same as `initial-route`) which is for testing.

Reply:
Keeping it visible might be useful for development/testing. Let's keep it as is.

Thank you again, JP!